Source/JavaScriptCore/runtime/JSFunction.cpp

```diff
@@ -516,6 +516,17 @@ String getCalculatedDisplayName(VM& vm, JSObject* object)
     return emptyString();
 }
 
+template<typename... StringTypes>
+String JSFunction::makeNameWithOutOfMemoryCheck(JSGlobalObject* globalObject, ThrowScope& throwScope, const char* messagePrefix, StringTypes... strings)
```

Comment:
Does this need to be a member function?  It doesn't look like it relies on any private / protected members of JSFunction.